### PR TITLE
For PR - #17932 Namerequest UI: Drop down menu does not work with keyboard input

### PR DESCRIPTION
**Pull request by jamespaologarcia**
*Issue #:* /bcgov/entity#17932

*Description of changes:*
Changed to v-list-item and updated v-list-item properties to align the elements.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-722-thpga0ef.web.app

File: src/assets/styles/base.scss [hunk 1]
```diff
@@ -76,7 +76,6 @@ html {
 .app-blue {
   color: $app-blue !important;
 }
-
```
Comment (severinbeauvais) [inline src/assets/styles/base.scss:79]: Most other classes in here are separated by a blank line, so please revert this deletion.

**Review by severinbeauvais — APPROVED** [review]
Great find! Looks good to me in Chrome and Firefox.

File: src/components/common/nested-select.vue [hunk 2]
```diff
@@ -158,6 +158,8 @@ export default class NestedSelect extends Vue {
 // Removes empty spaces on nested select when the list is collapsed
 ::v-deep .v-list-item {
   min-height: auto;
+  display: list-item;
+  list-style:none;
```
Comment (JazzarKarim) [inline src/components/common/nested-select.vue:162]: Please add a space after the ":".

**Review by JazzarKarim — APPROVED** [review]
LGTM James!

**Comment by severinbeauvais** [comment]
James, you merged it a little too quickly :)

There is no technical impact to not updating the app version (in package.json), but it's handy to see the version of a deployed app, ie:

image

Without this, it's harder to tell if your fix is present in the UI.

This doesn't need to be fixed IMHO.

**Pull request merged**